### PR TITLE
Switching task list load to application/hal+json

### PR DESCRIPTION
**Pull request by krishnan-aot**
# Issue Tracking

Issue Type: FEATURE

# Changes

Switches loading of task list from application/json to application/hal+json. The response of this API includes the count too saving one HTTP call (which is extremely amplified with websocket).

### CODE REVIEW

File: camunda-formio-tasklist-vue/src/services/axios.ts [hunk 1]
```diff
@@ -12,6 +12,18 @@ export function bpmAxios (bearerToken: string, CamundaUrl: string) {
   });
 }
 
+export function bpmAxiosWithHal (bearerToken: string, CamundaUrl: string) {
+  return axios.create({
+    baseURL: CamundaUrl,
+    withCredentials: false,
+    headers: {
+      "Access-Control-Allow-Origin": "*",
+      "Content-Type": "application/hal+json",
```
Comment (kurianbenoy-aot) [inline camunda-formio-tasklist-vue/src/services/axios.ts:21]: Isn't Accept to be changed to `application/hal+json`? When running the taskList, I am getting 415 error.

image

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.vscode/
```
Comment (kurianbenoy-aot) [inline .gitignore:1]: We already have a .gitignore inside folder, but it's not having the impact. Maybe you can delete the [current gitignore file](https://github.com/AOT-Technologies/forms-flow-ai-extensions/blob/master/camunda-formio-tasklist-vue/.gitignore) and modify the contents in file location you have shared now.

**Review by kurianbenoy-aot — CHANGES_REQUESTED** [review]
The tasklist is now not loading correctly after `application/hal+json` change.

File: camunda-formio-tasklist-vue/src/services/axios.ts (hunk 1 above)
Comment (krishnan-aot) [reply camunda-formio-tasklist-vue/src/services/axios.ts]: Corrected.

**Review by kurianbenoy-aot — APPROVED** [review]

**Review by kurianbenoy-aot — APPROVED** [review]
LGTM now

**Pull request merged**